openstates/md/committees.py

```diff
@@ -14,17 +14,24 @@ def scrape(self, term, chambers):
         doc.make_links_absolute(url)
 
         for a in doc.xpath('//a[contains(@href, "cmtepage")]'):
-            url = a.get('href').replace('stab=01', 'stab=04')
-            chamber = a.xpath('../../..//th/text()')[0]
+            url = a.get('href')
+            chamber_name = a.xpath('../../..//th/text()')[0]
+            if chamber_name == 'Senate Standing':
+                url = url.replace('stab=01', 'stab=04')
+            if chamber_name == 'House Standing':
+                url = url.replace('stab=01', 'stab=04')
             com_name = a.text
             if com_name is None:
                 continue
+            com_name = com_name.replace("Joint ", "")
+            com_name = com_name.replace("Committee on ", "")
+            com_name = com_name.replace(" Committee", "")
             com_name = com_name.strip()
-            if 'Senate' in chamber and 'upper' in chambers:
+            if 'Senate' in chamber_name:
                 chamber = 'upper'
-            elif 'House' in chamber and 'Delegation' not in chamber and 'lower' in chambers:
+            elif 'House' in chamber_name and 'Delegation' not in chamber_name:
                 chamber = 'lower'
-            elif chamber in ('Joint', 'Statutory', 'Special Joint', 'Other'):
+            elif 'Joint' or 'Statutory' or 'Special Joint' or 'Other' in chamber_name:
```

Comment:
This may be at the root of your problem; it'll always resolve to true, since everything between each `or` is its own boolean. `'Joint'` will resolve to `True`, for example, because it's a non-empty string. I've gotten bit by this one before!
```
if 'foo' or 'bar' in 'baz':
   print("This doesn't make sense!")
```